src/universal/.devcontainer/local-features/setup-user/install.sh

```diff
@@ -45,6 +45,8 @@ sudo_if() {
 # They are installed by the base image (python) which does not have the patch.
 sudo_if /usr/local/python/current/bin/python -m pip uninstall --yes setuptools
 sudo_if /usr/local/python/current/bin/python -m pip install --user --upgrade --no-cache-dir setuptools
+sudo_if /usr/local/python/3.9.16/bin/python -m pip uninstall --yes setuptools
```

Comment:
```suggestion
sudo_if /usr/local/python/3.9*/bin/python -m pip uninstall --yes setuptools
```

Can we use an `*`, so that every time we update the patch version, we don't have to update this file.

Reply:
Good point! Resolved with cf51ff32bba57123c486eac9b2208de5f10e643e.